src/main/java/org/shelter/service/PetShelter.java

```diff
@@ -0,0 +1,80 @@
+package org.shelter.service;
+
+import org.shelter.model.Pet;
+import org.shelter.model.PetShelterData;
+
+import java.util.Optional;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+public class PetShelter {
+    private Scanner scanner = new Scanner(System.in);
+    private PetShelterData data = PetJsonSerializer.readData();
+
+    public void run() {
+        System.out.println("Hello this is the " + data.getLabel());
+
+        while (true) {
+            System.out.println("Enter you command...");
+            String cmd = scanner.next();
+            switch (cmd) {
+                case "show" -> show();
+                case "add" -> add();
+                case "take" -> remove();
+                case "exit" -> exit();
+                default ->
+                        System.out.println("Unknown command, please try again. Available commands are: show, add, take, exit");
+
+            }
+        }
+    }
+
+    public void show() {
+        System.out.println("Here all our animals");
+        data.getPets().forEach(System.out::println);
+    }
+
+    public void add() {
+        System.out.println("name?");
+        String name = scanner.next();
+        System.out.println("breed?");
+        String breed = scanner.next();
+
+        int age = -1;
+        while (age < 0) {
+            try {
+                System.out.println("age?");
+                age = Integer.parseInt(scanner.next());
+            } catch (Exception e) {
+                // do nothing
```

Comment:
if it is an error, then do any action (throw an exception, log an error, ...), do not left it empty
also comment means nothing, so can be removed